### ISSUE TITLE
the accuracy is always 1.0000 while training

### ISSUE THREAD

**Issue by geroge-gao** [hx173149/C3D-tensorflow#59]
I want to train the C3D from the scratch,  but the accuracyis always 1.00000, I find some similar issues
#52 #36 #9  
I only change the gpu_num in the code.
here is my output:
Training Data    
accuracy: 1.00000
Validation Data Eval:
accuracy: 1.00000

is there a bug? or I should train for more time?

**Comment by kjunhwa** [comment]
hi
i have same accuracy like your results when i run train_c3d_ucf101.py.

do you know what kind of problem??
